### PR TITLE
Make update check configurable

### PR DESCRIPTION
**Pull request by l1ttleO**

### CODE REVIEW

File: resources/Changelog.txt [hunk 1]
```diff
@@ -289,8 +289,9 @@ v2.9:
     - Bump junit from 4.11 to 4.13.1 (@dependabot)
     - Fix syntax for "/temp..." commands (@l1ttleO)
     - Add timed subgroups (@l1ttleO, @ElgarL)
-    - Add PAPI support...
+    - Add PAPI support (@ElgarL)
```
Comment (ElgarL) [inline resources/Changelog.txt:292]: Lets not put a credit on everything we add to the Changelog. Credits on new Language additions or large features is fine else we are quickly going to spam names in what should be informational.

**Review by ElgarL — COMMENTED** [review]
I'm not sure about allowing the update check be configurable.

**Review by ElgarL — APPROVED** [review]

**Comment by ElgarL** [comment]
Still have to decide if we want this setting to be changeable.
As the GM config rarely changes once its set to off its likely never going to change again.

**Comment by l1ttleO** [comment]
Closing:
a) Inactivity
b) I have to re-do it because I resolved conflicts incorrectly

**Pull request closed without merging**